index.bs

```diff
@@ -3183,10 +3183,12 @@ associated with or [=scoped=] to, respectively.
                 on {{PublicKeyCredentialEntity/name}}'s value prior to displaying the value to the user or
                 including the value as a parameter of the [=authenticatorMakeCredential=] operation.
 
-          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a [=human palatability|human-palatable=] identifier for a
-            [=user account=]. It is intended only for display, i.e., aiding the user in determining the difference between user
-            accounts with similar {{PublicKeyCredentialUserEntity/displayName}}s. For example, "alexm", "alex.mueller@example.com"
-            or "+14255551234".
+          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a
+            [=human palatability|human-palatable=] identifier for a [=user account=]. This
+            identifier is the primary value displayed to users by [=Clients=] to help users
+            understand with which [=user account=] a credential is associated. Examples of suitable
+            values for this identifier include, "alexm", "alex.mueller@example.com", or
+            "+14255551234".
```

Comment:
This section needs to be tweaked to more strongly emphasize that this is **the** value for users to distinguish multiple credentials that may have the same email address (thinking about multi-tenant environments.)

I also want to add an example of something like "alex.mueller@example.com (production)" to open up perception to possible ways of differentiating multi-tenant credentials.

> [=Clients=] may display this value to help users differentiate between multiple
        [=credentials=] with the same value of {{PublicKeyCredentialEntity/name}}.

I should also remove this from the section **below**.

Reply:
Example from @timcappalli:

```
dn: Tim Cappalli (Microsoft)
n: tim.cappalli@microsoft.com

is a common pattern I see
```

Reply:
I took another stab at the wording here to try and emphasize that RP's should add to `name` to differentiate multiple credentials when otherwise all of them would end up with the same value.